src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -113,13 +113,17 @@ internal static int Start(string bannerText, string helpText, string[] args)
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Pwsh startup and PATH.pwsh starts even if PATH is not defined
```powershell
Expected strings to be the same, but they were different.
Expected length: 52
Actual length:   700
Strings differ at index 52.
Expected: '...lish:'
But was:  '...lish:/User...'
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 1009
1009:         $path | Should -BeExactly ($PSHOME + [System.IO.Path]::PathSeparator)
```